prompts/Dutch (Netherlands)_nl-NL/0000000001_0300000050_General.txt

```diff
@@ -155,13 +155,13 @@
 0000000155	Tegenwoordig maken we veel gebruik van onderwaterrobots en bodemlanders.
 0000000156	Dat bevestigt de hekkensluiter van de Eredivisie.
 0000000157	Dat gebeurde laat in de nacht.
-0000000158	Volgens mij mogen we maar een paar meenemen. 
+0000000158	Volgens mij mogen we maar een paar meenemen.
 0000000159	Drie procent in tweeduizend zeventien.
 0000000160	De middenvelder wordt overgenomen van Zenit Sint-Petersburg.
 0000000161	Op de nieuwe locatie wordt een geheel nieuw pand van circa vierduizendtweehonderd vierkante meter neergezet.
 0000000162	Zij kregen onvoldoende steun in de raad.
 0000000163	Het aantal gebeurtenissen zorgde voor veel hinder voor reizigers.
-0000000164	De laatste keer dat zij die titel won was in tweeduizend zes.
+0000000164	De laatste keer dat zij die titel won was tweeduizend zes.
```

Comment:
This one is debatable. Considering this has to be read out (and thus speaking terms), I think it makes more sense to revert this. As she won title somewhere in 2006.

Reply:
Reading this again, I agree.